### PR TITLE
feat: Allow users to override secure boot key and password

### LINKED ISSUE #32: Support action usage outside UBlue
It would be good to expose options like secure boot keys so the action can be used by others to build ISOs in their personal images.

Right now, the only hard-coded ublueified configuration I could find (and cannot be overwritten) is for secure boot cert enrollment.

### PR DESCRIPTION
**Pull request by noelmiller**
# Purpose

This PR is allowing a user to override the secure boot key and secure boot password to allow them to enroll their own key.

Fixes: #32 

# What I did
- Updated tests to include a test for secure-boot ISOs
- Removed unneeded hardcoded files
- Added Variables for SECURE_BOOT_KEY_URL and ENROLLMENT_PASSWORD
- Added checks to pull down secure boot key from SECURE_BOOT_KEY_URL

### CODE REVIEW

**Comment by noelmiller** [comment]
crap, not ready for review. disregard.

**Comment by noelmiller** [comment]
OK ready for review. For now I am going to stick with a URL since the primary purpose of enrolling a key is to allow our kmods to work. Most users will not require this level of customization and would use our key anyway if they were building off any of our base images with our kmods.

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,8 @@ IMAGE_TAG = $(VERSION)
 EXTRA_BOOT_PARAMS =
 VARIANT = Kinoite
 WEB_UI = false
+ENROLLMENT_PASSWORD = isogenerator
```
Comment (bsherman) [inline Makefile]: I know I suggested "isogenerator" as a more generic default password... but this IS a ublue org action... 

Should it default to "ublue-os" for the sanity of defaulting to what we've put in all our docs?

(More a question for the team)

Comment (noelmiller) [reply Makefile]: It would save effort in our actions as I wouldn't have to specify it every time.

Comment (bsherman) [reply Makefile]: @KyleGospo do you agree `ublue-os` is a better default?

If so, lets change that back as the default.  Please forgive me for causing confusion. And I'll approve then. :-)

Comment (p5) [reply Makefile]: The rest of the params are defaulting to ublue values, so I vote to keep ublue-os by default.

Comment (bsherman) [reply Makefile]: @noelmiller go for it!

**Review by bsherman — APPROVED** [review]
I think it looks good!

**Review by KyleGospo — APPROVED** [review]

**Pull request merged**